package.json

```diff
@@ -34,7 +35,7 @@
     "file"
   ],
   "scripts": {
-    "test": "nyc mocha ./test",
+    "test": "nyc mocha --recursive --exit",
```

Comment:
fwiw, `--recursive` shouldn't be needed as there is no nested folders under [test/](https://github.com/adobe/node-httptransfer/tree/master/test). (It doesn't hurt though).

Reply:
Just in case in the future and to keep repos uniform where possible